src/app/f2f/controllers/checkDetails.test.js

```diff
@@ -76,8 +76,40 @@ describe("CheckDetails controller", () => {
         await checkDetailsController.locals(req, res, next);
       });
 
-      it("should call callback with error and existing locals", () => {
+      it("should call callback with error and existing locals - first Post Office option", () => {
+        req.sessionModel.set("countryCode", "GBR");
+        req.sessionModel.set("country", "United Kingdom");
+        const postOfficeName = req.sessionModel.get("postOfficeName");
+        const postOfficePostcode = req.sessionModel.get("postOfficePostcode");
+        const postCodeLat = req.sessionModel.get("postOfficeLatitude");
+        const postCodeLong = req.sessionModel.get("postOfficeLongitude");
+        const countryCode = req.sessionModel.get("countryCode");
+        const country = req.sessionModel.get("country");
+
+        expect(next).to.have.been.calledWith(error, superLocals);
+        expect(postOfficeName).to.equal("BRANCH NAME");
+        expect(postOfficePostcode).to.equal("G41 1ED");
+        expect(postCodeLat).to.equal("100000");
+        expect(postCodeLong).to.equal("-100000");
+        expect(countryCode).to.equal("GBR");
+        expect(country).to.equal("United Kingdom");
+        
+      });
```

Comment:
Do we want a test for the second option?

Also, can I ask why the test for the first option is so different from the ones for the third and last?